widgetastic_manageiq/__init__.py

```diff
@@ -1327,7 +1327,7 @@ def pages(self):
                     self.next_page()
 
         else:
-            return
```

Comment:
can you create new PR with this small fix ? This difference in behavior breaks a bunch of tests.